djstripe/models.py

```diff
@@ -26,6 +27,7 @@
 from .signals import subscription_made, cancelled, card_changed
 from .signals import webhook_processing_error
 
+_log = logging.getLogger(__name__)
```

Comment:
While logging is great, we're not looking to include logging statements in this project at this time.